test/e2e-serving-upgrade-tests.sh

```diff
@@ -69,6 +66,12 @@ function test_setup() {
   cd ${OPERATOR_DIR}
 }
 
+# Create test namespaces for serving
+function create_test_namespace_serving() {
+  kubectl get ns ${SERVING_TESTS_NAMESPACE} || kubectl create namespace ${SERVING_TESTS_NAMESPACE}
+  kubectl get ns ${SERVING_TESTS_ALT_NAMESPACE} || kubectl create namespace ${SERVING_TESTS_ALT_NAMESPACE}
```

Comment:
These need the correct labels for the configmap validation webhooks to function

https://github.com/knative/serving/blob/main/config/core/webhooks/configmap-validation.yaml#L30-L33